src/blurb/blurb.py

```diff
@@ -1013,7 +1006,7 @@ def print(*a, sep=" "):
             assert len(filenames) == 1
             blurbs.load(filenames[0])
 
-        header = "What's New in Python " + printable_version(version) + "?"
+        header = printable_version(version)
```

Comment:
Something (I'm guessing this line?) is dropping "Python" before the version numbers in the headers, which has a knock-on effect that a lot (but not all) of anchors change from like `#python-3-13-0-alpha-4` to `#id62`:

(`main` on left, PR on right)

<img width="1582" alt="image" src="https://github.com/user-attachments/assets/059812a2-fbe8-44b0-8464-a856ef2ffb14" />

<img width="1582" alt="image" src="https://github.com/user-attachments/assets/8a1f6af4-f7aa-49d1-9c0b-319b7a8396d0" />